src/messaging/rabbitmq/conn.ts

```diff
@@ -11,7 +11,7 @@ export default class RMQConnection {
 
   public async getConnection(): Promise<Connection> {
     this.conn = await amqp.connect({
-      protocol: 'amqp',
+      protocol: process.env.RMQ_PROTOCOL || 'amqp',
       hostname: process.env.RMQ_HOST,
       port: process.env.RMQ_PORT ? parseInt(process.env.RMQ_PORT) : 5672,
```

Comment:
default fallback should be 15672 and not 5672 .. no?

Reply:
line no. 16

Reply:
5672 is the port for connecting to broker for publishing message, 15672 is management API port